src/scene/vcModel.cpp

```diff
@@ -281,6 +292,50 @@ void vcModel::OnNodeUpdate(vcState *pProgramState)
     }
   }
 
+  if (vdkProjectNode_GetMetadataDouble(m_pNode, "visualization.GPSTime.minTime", &m_visualization.GPSTime.minTime, pProgramState->settings.visualization.GPSTime.minTime) != vE_Success && vdkProjectNode_GetMetadataDouble(m_pNode, "visualization.GPSTime.maxTime", &m_visualization.GPSTime.maxTime, pProgramState->settings.visualization.GPSTime.maxTime) != vE_Success)
+  {
+    const char *pGPSTime = m_metadata.Get("AttrMinMax_udGPSTime").AsString();
```

Comment:
`m_metadata` is deprecated to be removed, you'll need to use the project node versions.